core/libp2p/swarm/swarm.hpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_SWARM_HPP
+#define KAGOME_SWARM_HPP
+
+#include <boost/signals2.hpp>
+#include <rxcpp/rx-observable.hpp>
+#include "libp2p/common_objects/peer_info.hpp"
+#include "libp2p/connection/connection.hpp"
+#include "libp2p/connection/connection_status.hpp"
+
+namespace libp2p {
+  namespace swarm {
+    /**
+     * Establishes connections with other peers
+     */
+    class Swarm {
```

Comment:
should be named Switch 
https://github.com/libp2p/js-libp2p-switch#table-of-contents

https://github.com/libp2p/js-libp2p-swarm was renamed to https://github.com/libp2p/js-libp2p-switch